packages/appstore/src/constants/trading-hub-content.tsx

```diff
@@ -13,39 +14,67 @@ type TTradingHubContents = Record<'step1' | 'step2' | 'step3' | 'step4' | 'step5
 
 export const trading_hub_contents: TTradingHubContents = {
     step1: {
-        component: 'Step1',
+        component: <StaticDashboard />,
         footer_header: localize('Welcome to Trading Hub'),
         footer_text: localize('This is your new dashboard'),
         has_next_content: false,
     },
     step2: {
-        component: 'Step2',
+        component: <StaticDashboard is_cfd_text_blurry is_options_text_blurry is_platformlauncher_blurry />,
         footer_header: localize('Trading accounts'),
         footer_text: localize(
             'These are the trading accounts available to you. You can click on an account to find out more.'
         ),
         has_next_content: false,
     },
     step3: {
-        component: 'Step3',
+        component: <StaticDashboard is_cfd_item_blurry is_options_item_blurry is_platformlauncher_blurry />,
         footer_header: localize('CFDs, Options or Multipliers'),
         footer_text: localize('You can choose between CFD trading accounts or Options and Multipliers accounts'),
         has_next_content: false,
     },
     step4: {
-        component: 'Step4',
+        component: (
+            <StaticDashboard is_cfd_text_blurry is_options_text_blurry is_platformlauncher_blurry is_button_animated />
+        ),
         footer_header: localize('‘Get’ the accounts you want'),
         footer_text: localize('Click the ‘Get’ button to create an account'),
         has_next_content: false,
     },
     step5: {
-        component: 'Step5',
+        component: (
+            <StaticDashboard
+                is_grey
+                has_account
+                currency={'USD'}
+                derived_amount={'369'}
+                loginid={'20884385'}
+                is_trade_blurry
+                is_topup_animated
+                is_cfd_text_blurry
+                is_options_text_blurry
+                has_applauncher_account
+            />
+        ),
         footer_header: localize('Top-up your account'),
         footer_text: localize('Once you have an account click ‘Top-up’ to add funds '),
         has_next_content: false,
     },
     step6: {
-        component: 'Step6',
```

Comment:
1. Just the `Derived` trade button should be highlighted in CFD part( not the Financial)
2. More Derived accounts and More Financial accounts should be added
3. In the Options part the DTrader platform should be highlighted and has animation( others should be lowlighted)
4.Add options button should be added at the right side of the `Options accounts`

Test Link:
![image](https://user-images.githubusercontent.com/82078941/192284661-5bd8144b-89ab-450a-9513-ef5912df6f17.png)

Design:
![image](https://user-images.githubusercontent.com/82078941/192284621-85c84a78-c830-4138-a429-90821af6fedf.png)